bridge/contracts/Bridge.sol

```diff
@@ -476,7 +478,7 @@ contract Bridge is Initializable, IBridge, IERC777Recipient, UpgradablePausable,
     }
 
     function changeSideTokenFactory(address newSideTokenFactory) external onlyOwner {
-        require(newSideTokenFactory != NULL_ADDRESS, "Bridge: SideTokenFactory is empty");
+        require(newSideTokenFactory != NULL_ADDRESS, "Bridge: SideTokenFactory empty");
```

Comment:
if we remove the 'is', maybe we can change it to 'empty SideTokenFactory'? I think it makes more sense

Reply:
Definitely, done at https://github.com/rsksmart/tokenbridge/pull/242/commits/f03f3e2cdeaf0ebe20f1376e9269dc45070c7034